opentelemetry/src/global/internal_logging.rs

```diff
@@ -50,11 +50,30 @@ macro_rules! otel_warn {
         {
             tracing::warn!(name: $name, target: env!("CARGO_PKG_NAME"), "");
         }
+        #[cfg(not(feature = "internal-logs"))]
```

Comment:
Agree with this change, but for future - please make them into own separate PR to make it easier to review.